File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Bass/HOTCODEMASTERGetGrdListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Bass
{
    [Serializable]
    public class HOTCODEMASTERGetGrdListInfo
    {
        private String _hotCode;
        private String _hotCode7;
        private String _cinCode;
        private String _dispenseCode;
        private String _logisticCode;
        private String _janCode;
        private String _yakKijunCode;
        private String _yjCode;
        private String _sgCode;
        private String _sgCode1;
        private String _hangmogName;
        private String _hangmogName1;
        private String _hangmogName2;
        private String _standardUnit;
        private String _pkgStatus;
        private String _pkgNumUnit;
        private String _ordDanui;
        private String _pkgTotal;
        private String _pkgTotalUnit;
        private String _clsif;
        private String _manufComp;
        private String _salesComp;
        private String _clsifUpd;
        private String _sgYmd;

        public String HotCode
        {
            get { return this._hotCode; }
            set { this._hotCode = value; }
        }

        public String HotCode7
        {
            get { return this._hotCode7; }
            set { this._hotCode7 = value; }
        }

        public String CinCode
        {
            get { return this._cinCode; }
            set { this._cinCode = value; }
        }

        public String DispenseCode
        {
            get { return this._dispenseCode; }
            set { this._dispenseCode = value; }
        }

        public String LogisticCode
        {
            get { return this._logisticCode; }
            set { this._logisticCode = value; }
        }

        public String JanCode
        {
            get { return this._janCode; }
            set { this._janCode = value; }
        }

        public String YakKijunCode
        {
            get { return this._yakKijunCode; }
            set { this._yakKijunCode = value; }
        }

        public String YjCode
        {
            get { return this._yjCode; }
            set { this._yjCode = value; }
        }

        public String SgCode
        {
            get { return this._sgCode; }
            set { this._sgCode = value; }
        }

        public String SgCode1
        {
            get { return this._sgCode1; }
            set { this._sgCode1 = value; }
        }

        public String HangmogName
        {
            get { return this._hangmogName; }
            set { this._hangmogName = value; }
        }

        public String HangmogName1
        {
            get { return this._hangmogName1; }
            set { this._hangmogName1 = value; }
        }

        public String HangmogName2
        {
            get { return this._hangmogName2; }
            set { this._hangmogName2 = value; }
        }

        public String StandardUnit
        {
            get { return this._standardUnit; }
            set { this._standardUnit = value; }
        }

        public String PkgStatus
        {
            get { return this._pkgStatus; }
            set { this._pkgStatus = value; }
        }

        public String PkgNumUnit
        {
            get { return this._pkgNumUnit; }
            set { this._pkgNumUnit = value; }
        }

        public String OrdDanui
        {
            get { return this._ordDanui; }
            set { this._ordDanui = value; }
        }

        public String PkgTotal
        {
            get { return this._pkgTotal; }
            set { this._pkgTotal = value; }
        }

        public String PkgTotalUnit
        {
            get { return this._pkgTotalUnit; }
            set { this._pkgTotalUnit = value; }
        }

        public String Clsif
        {
            get { return this._clsif; }
            set { this._clsif = value; }
        }

        public String ManufComp
        {
            get { return this._manufComp; }
            set { this._manufComp = value; }
        }

        public String SalesComp
        {
            get { return this._salesComp; }
            set { this._salesComp = value; }
        }

        public String ClsifUpd
        {
            get { return this._clsifUpd; }
            set { this._clsifUpd = value; }
        }

        public String SgYmd
        {
            get { return this._sgYmd; }
            set { this._sgYmd = value; }
        }

        public HOTCODEMASTERGetGrdListInfo() { }

        public HOTCODEMASTERGetGrdListInfo(String hotCode, String hotCode7, String cinCode, String dispenseCode, String logisticCode, String janCode, String yakKijunCode, String yjCode, String sgCode, String sgCode1, String hangmogName, String hangmogName1, String hangmogName2, String standardUnit, String pkgStatus, String pkgNumUnit, String ordDanui, String pkgTotal, String pkgTotalUnit, String clsif, String manufComp, String salesComp, String clsifUpd, String sgYmd)
        {
            this._hotCode = hotCode;
            this._hotCode7 = hotCode7;
            this._cinCode = cinCode;
            this._dispenseCode = dispenseCode;
            this._logisticCode = logisticCode;
            this._janCode = janCode;
            this._yakKijunCode = yakKijunCode;
            this._yjCode = yjCode;
            this._sgCode = sgCode;
            this._sgCode1 = sgCode1;
            this._hangmogName = hangmogName;
            this._hangmogName1 = hangmogName1;
            this._hangmogName2 = hangmogName2;
            this._standardUnit = standardUnit;
            this._pkgStatus = pkgStatus;
            this._pkgNumUnit = pkgNumUnit;
            this._ordDanui = ordDanui;
            this._pkgTotal = pkgTotal;
            this._pkgTotalUnit = pkgTotalUnit;
            this._clsif = clsif;
            this._manufComp = manufComp;
            this._salesComp = salesComp;
            this._clsifUpd = clsifUpd;
            this._sgYmd = sgYmd;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/bass/HOTCODEMASTERGetGrdListInfo.java
package nta.med.data.model.ihis.bass;

import java.io.Serializable;
import java.util.Date;

public class HOTCODEMASTERGetGrdListInfo implements Serializable{
	private String hotCode          ; 
	private String hotCode7         ;
	private String cinCode          ;
	private String dispenseCode     ;
	private String logisticCode     ;
	private String janCode          ;
	private String yakKijunCode    ;
	private String yjCode           ;
	private String sgCode           ;
	private String sgCode1           ;
	private String hangmogName       ;
	private String hangmogName1      ;
	private String hangmogName2      ;
	private String standardUnit      ;
	private String pkgStatus         ;
	private String pkgNumUnit       ;
	private String ordDanui          ;
	private String pkgTotal          ;
	private String pkgTotalUnit     ;
	private String clsif             ;
	private String manufComp         ;
	private String salesComp         ;
	private String clsifUpd          ;
	private Date sgYmd             ;
	public HOTCODEMASTERGetGrdListInfo(String hotCode, String hotCode7,
			String cinCode, String dispenseCode, String logisticCode,
			String janCode, String yakKijunCode, String yjCode, String sgCode,
			String sgCode1, String hangmogName, String hangmogName1,
			String hangmogName2, String standardUnit, String pkgStatus,
			String pkgNumUnit, String ordDanui, String pkgTotal,
			String pkgTotalUnit, String clsif, String manufComp,
			String salesComp, String clsifUpd, Date sgYmd) {
		super();
		this.hotCode = hotCode;
		this.hotCode7 = hotCode7;
		this.cinCode = cinCode;
		this.dispenseCode = dispenseCode;
		this.logisticCode = logisticCode;
		this.janCode = janCode;
		this.yakKijunCode = yakKijunCode;
		this.yjCode = yjCode;
		this.sgCode = sgCode;
		this.sgCode1 = sgCode1;
		this.hangmogName = hangmogName;
		this.hangmogName1 = hangmogName1;
		this.hangmogName2 = hangmogName2;
		this.standardUnit = standardUnit;
		this.pkgStatus = pkgStatus;
		this.pkgNumUnit = pkgNumUnit;
		this.ordDanui = ordDanui;
		this.pkgTotal = pkgTotal;
		this.pkgTotalUnit = pkgTotalUnit;
		this.clsif = clsif;
		this.manufComp = manufComp;
		this.salesComp = salesComp;
		this.clsifUpd = clsifUpd;
		this.sgYmd = sgYmd;
	}
	public String getHotCode() {
		return hotCode;
	}
	public void setHotCode(String hotCode) {
		this.hotCode = hotCode;
	}
	public String getHotCode7() {
		return hotCode7;
	}
	public void setHotCode7(String hotCode7) {
		this.hotCode7 = hotCode7;
	}
	public String getCinCode() {
		return cinCode;
	}
	public void setCinCode(String cinCode) {
		this.cinCode = cinCode;
	}
	public String getDispenseCode() {
		return dispenseCode;
	}
	public void setDispenseCode(String dispenseCode) {
		this.dispenseCode = dispenseCode;
	}
	public String getLogisticCode() {
		return logisticCode;
	}
	public void setLogisticCode(String logisticCode) {
		this.logisticCode = logisticCode;
	}
	public String getJanCode() {
		return janCode;
	}
	public void setJanCode(String janCode) {
		this.janCode = janCode;
	}
	public String getYakKijunCode() {
		return yakKijunCode;
	}
	public void setYakKijunCode(String yakKijunCode) {
		this.yakKijunCode = yakKijunCode;
	}
	public String getYjCode() {
		return yjCode;
	}
	public void setYjCode(String yjCode) {
		this.yjCode = yjCode;
	}
	public String getSgCode() {
		return sgCode;
	}
	public void setSgCode(String sgCode) {
		this.sgCode = sgCode;
	}
	public String getSgCode1() {
		return sgCode1;
	}
	public void setSgCode1(String sgCode1) {
		this.sgCode1 = sgCode1;
	}
	public String getHangmogName() {
		return hangmogName;
	}
	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}
	public String getHangmogName1() {
		return hangmogName1;
	}
	public void setHangmogName1(String hangmogName1) {
		this.hangmogName1 = hangmogName1;
	}
	public String getHangmogName2() {
		return hangmogName2;
	}
	public void setHangmogName2(String hangmogName2) {
		this.hangmogName2 = hangmogName2;
	}
	public String getStandardUnit() {
		return standardUnit;
	}
	public void setStandardUnit(String standardUnit) {
		this.standardUnit = standardUnit;
	}
	public String getPkgStatus() {
		return pkgStatus;
	}
	public void setPkgStatus(String pkgStatus) {
		this.pkgStatus = pkgStatus;
	}
	public String getPkgNumUnit() {
		return pkgNumUnit;
	}
	public void setPkgNumUnit(String pkgNumUnit) {
		this.pkgNumUnit = pkgNumUnit;
	}
	public String getOrdDanui() {
		return ordDanui;
	}
	public void setOrdDanui(String ordDanui) {
		this.ordDanui = ordDanui;
	}
	public String getPkgTotal() {
		return pkgTotal;
	}
	public void setPkgTotal(String pkgTotal) {
		this.pkgTotal = pkgTotal;
	}
	public String getPkgTotalUnit() {
		return pkgTotalUnit;
	}
	public void setPkgTotalUnit(String pkgTotalUnit) {
		this.pkgTotalUnit = pkgTotalUnit;
	}
	public String getClsif() {
		return clsif;
	}
	public void setClsif(String clsif) {
		this.clsif = clsif;
	}
	public String getManufComp() {
		return manufComp;
	}
	public void setManufComp(String manufComp) {
		this.manufComp = manufComp;
	}
	public String getSalesComp() {
		return salesComp;
	}
	public void setSalesComp(String salesComp) {
		this.salesComp = salesComp;
	}
	public String getClsifUpd() {
		return clsifUpd;
	}
	public void setClsifUpd(String clsifUpd) {
		this.clsifUpd = clsifUpd;
	}
	public Date getSgYmd() {
		return sgYmd;
	}
	public void setSgYmd(Date sgYmd) {
		this.sgYmd = sgYmd;
	}

}
